.travis.yml

```diff
@@ -0,0 +1,40 @@
+dist: xenial
+
+lang: go
+
+go:
+  - 1.7.x
+
+jdk:
+  - oraclejdk8
+
+env:
+  - BAZEL_VERSION=0.4.2
+
+addons:
+  apt:
+    packages:
+      - wget
+      - uuid-dev
+      - uuid-runtime
+
+cache:
+  directories:
+    - $HOME/bazel/install
+    - $HOME/bazel/outbase
+
+before_install:
+  - mkdir -p ${HOME}/bazel/install
+  - cd ${HOME}/bazel/install
+  - wget --no-clobber "https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel_${BAZEL_VERSION}-linux-x86_64.deb" 
+  - chmod +x bazel_${BAZEL_VERSION}-linux-x86_64.deb
+  - sudo dpkg -i bazel_${BAZEL_VERSION}-linux-x86_64.deb
+  - cd ${TRAVIS_BUILD_DIR}
+  - mv .bazelrc .bazelrc.orig
+  - cat .bazelrc.travis .bazelrc.orig > .bazelrc
+
+script:
+  - bazel --output_base=${HOME}/bazel/outbase test //...
```

Comment:
I usually run //contrib/endpoints/src/...

I am not sure if //... works.  

It did not work in my local workspace